doc/rbd/rbd-mirroring.rst

```diff
@@ -289,6 +294,16 @@ For example::
 .. tip:: You can enable journaling on all new images by default by adding
    ``rbd default features = 125`` to your Ceph configuration file.
 
+.. tip:: ``rbd-mirror`` tunables are set by default to values suitable for
+   mirroring an entire pool.  When using ``rbd-mirror` to migrate single
+   volumes been clusters you may realize substantial performance gains
```

Comment:
"When using ``rbd-mirror`` to migrate single volumes been clusters you may realize substantial performance gains by setting [string] and [string] within the ``[client]`` config section of the local or centralized configuration." kind of breaks down, syntactically, starting with "been clusters you may realize". I'm not really sure what's intended here.

Reply:
`realize` as a synomym for `achieve`.